charts/kueue/values.yaml

```
@@ -143,3 +143,24 @@ webhookService:
       protocol: TCP
       targetPort: 9443
   type: ClusterIP
+
+# kueue-viz dashboard
+kueueViz:
+  namespace: kueue-system
+  enabled: true
```

```suggestion
  enabled: false
```
Should it be disabled by default?

---

disabled by default